aea/cli/utils/package_utils.py

```diff
@@ -489,3 +525,30 @@ def try_get_balance(  # pylint: disable=unused-argument
         return balance
     except ValueError as e:  # pragma: no cover
         raise click.ClickException(str(e))
+
+
+def get_latest_version_available_in_registry(
```

Comment:
This should be in registry utils not here...

Reply:
done